papis/paths.py

```diff
@@ -88,14 +88,24 @@ def normalize_path(path: str, *,
     else:
         regex_pattern = fr"[^a-zA-Z0-9.{extra_chars}]+"
 
+    # fix slugify forcefully replacing hyphens
+    if "-" in extra_chars:
+        placeholder = "slugifyhyphenfixer"
```

Comment:
Make this a global variable `_SLUGIFY_HYPHEN_PLACEHOLDER = "..."`?

Reply:
> don't think anyone would need to modify this, but sure.

The leading underscore should discourage people from accessing it. My thought was just that it's a global variable, so we might as well make it look like one.

> though i struggle to understand where vars are stored in the project

Just meant to put it on top of the function here (there's no special place).